moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -409,19 +408,39 @@ bool IKFastKinematicsPlugin::initialize(const moveit::core::RobotModel& robot_mo
   }
 
   storeValues(robot_model, group_name, base_frame, tip_frames, search_discretization);
+  if (!lookupParam("link_prefix", link_prefix_, std::string("")))
+  {
+    ROS_INFO_NAMED(name_, "link_prefix parameter not set. Assuming empty prefix.");
+  }
+  else
+  {
+    ROS_INFO_STREAM_NAMED(name_, "using link_prefix " << link_prefix_);
+  }
+
+  // verbose error output. subsequent checks in computeRelativeTransform return false then
```

Comment:
If you add these error messages here, the ones from `computeRelativeTransform()` should be suppressed.
I have implemented this in https://github.com/ubi-agni/moveit/tree/ikfast-prefix

Reply:
Thanks! I've merged your branch in here.